frameworks/Rust/actix/Cargo.toml

```
@@ -44,6 +44,8 @@ bytes = "0.5.3"
 num_cpus = "1.0"
 futures = "0.3.1"
 http = "0.2"
+simd-json = { git = "https://github.com/simd-lite/simd-json.git" }
```

This should work with the crate :) it's compatible with the derive

```suggestion
simd-json = "0.3"
```